include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3270,6 +3270,22 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
 
 namespace internal
 {
+  template <typename VectorType, bool>
+  struct ConstBlockVectorSelector
+  {};
```

Comment:
You might as well remove the whole body of this class and just provide a forward declaration since there is no case you are not covering via a specialization.

Reply:
Thanks for the review. I have removed the body (I just did it how it was below).